packages/reactotron-core-ui/src/timeline-command/timeline-command.tsx

```diff
@@ -30,8 +30,15 @@ export class TimelineCommand extends React.Component<Props, State> {
     const PreviewComponent = timelineConfig.preview
     const BodyComponent = timelineConfig.component
 
+    // Ok. So this is awful. I know there is a better way of handling this
+    // with tailwind and flexbox, but I have no idea what it is.
+    // So for the minute, please embrace the hack. Sorry!
+    const style = {
```

Comment:
Yeah, I plan on trying to get more time to get this base component worked out/styled better since we are getting actual commands in. I will see if I can figure something out here.

Reply:
I'm new to tailwind too... maybe the right approach is to have the body scroll naturally and the tabs and such be pinned.  I dunno.  We'll get it.